terraform/cloud-platform-components/nginx-ingress-acme.tf

```diff
@@ -30,7 +30,7 @@ controller:
 
   service:
     annotations:
-      external-dns.alpha.kubernetes.io/hostname: "*.apps.${data.terraform_remote_state.cluster.cluster_domain_name},apps.${data.terraform_remote_state.cluster.cluster_domain_name}"
+      external-dns.alpha.kubernetes.io/hostname: "${terraform.workspace == local.live_workspace ? format("*.%s, *.apps.%s, apps.%s", local.live_domain, data.terraform_remote_state.cluster.cluster_domain_name, data.terraform_remote_state.cluster.cluster_domain_name) : format("*.apps.%s, apps.%s", data.terraform_remote_state.cluster.cluster_domain_name, data.terraform_remote_state.cluster.cluster_domain_name)}"
```

Comment:
I find this a bit hard to parse, can't we instead do something like:
```
"*.apps.${data.terraform_remote_state.cluster.cluster_domain_name},apps.${data.terraform_remote_state.cluster.cluster_domain_name}${terraform.workspace == local.live_workspace ? format(",*.%s", local.live_domain) : ""}"
```

Reply:
neat! changed